sensor_msgs/Image im
string node_name
int16 image_number
